### ISSUE TITLE
Cycle/repeat tweets on wall

### ISSUE THREAD

**Issue by aneeshd16** [fossasia/loklak_webclient#377]
The search results are sorted by created_at. The most recent one is at the top. Currently the following happens:
* a sorted array of tweets is maintained (`nextStatuses`). The most recent tweet is the first tweet.
* The initial search results are sorted by default, so `nextStatuses` becomes the same as the search results array.
* Another function (`$interval`) gets the first tweet from `nextStatuses`, displays it and removes it from `nextStatuses` after every 4 seconds.
* Whenever a new search happens, the search results are added to `nextStatuses` (provided the tweet_id has not already been processed), and `nextStatuses` is sorted. In this way, whenever the `$interval` function runs, it will display the latest tweet, thus giving it real-time functionality.

In short, most recent tweets are shown first.

Lets say the initial search results are as follows:
````
t1
t2
t3
t4
````
(`t1` is the latest tweet).
Therefore, `nextStatuses` will be same. ie:
````
t1
t2
t3
t4
````
The `$interval` functions displays t1 and removes it from `nextStatuses`. `nextStatuses` is now:
````
t2
t3
t4
````
At this point, a new search occurs and search results are:
````
t5
t1
t2
t3
t4
````
(`t5` is the newest tweet).
Thererfore, according to the logic `nextStatuses` will become:
````
t5
t2
t3
t4
````
(Since `t1` has already been displayed, it will not be shown again).
Now, `t5` gets shown before the other older tweets.

This will keep the cycle going.

**Comment by aneeshd16** [comment]
Objectives:
If cycle is activated please give options 
* [ ] cycle delay time
* [ ] cycle post limit


![screenshot from 2015-08-02 23 30 19](https://cloud.githubusercontent.com/assets/1583873/9028132/9aade3c2-396e-11e5-864d-095e6c074349.png)

* [ ] If cycle is deactivated do not show options
![screenshot from 2015-08-02 23 30 05](https://cloud.githubusercontent.com/assets/1583873/9028141/c5ce3606-396e-11e5-974d-8b354cd62ed4.png)

**Comment by sopu** [comment]
@aneeshd16 why is this issue still open? 
There are several such issues on which a lot of work has been done but have not been closed. IMO these should be closed and the new issues should be opened mentioning only the unfinished part of the feature/bug/design-check. 
It would be easier to address them.
Cheers!!
